Build/Scripts/runTests.sh

```diff
@@ -164,149 +262,141 @@ Options:
         Send xdebug information to a different port than default 9003 if an IDE like PhpStorm
         is not listening on default port.
 
-    -z <xdebug modes>
-        Only with -x and -s functional|unit|acceptance
-        This sets the used xdebug modes. Defaults to 'debug,develop'
-
     -n
-        Only with -s cgl
+        Only with -s cgl|composerNormalize
         Activate dry-run in CGL check that does not actively change files and only prints broken ones.
 
     -u
-        Update existing ${IMAGE_PREFIX}core-testing-*:latest docker images. Maintenance call to docker pull latest
-        versions of the main php images. The images are updated once in a while and only the youngest
-        ones are supported by core testing. Use this if weird test errors occur. Also removes obsolete
-        image versions of ${IMAGE_PREFIX}core-testing-*.
-
-    -v
-        Enable verbose script output. Shows variables and docker commands.
+        Update existing typo3/core-testing-*:latest container images and remove dangling local volumes.
+        New images are published once in a while and only the latest ones are supported by core testing.
+        Use this if weird test errors occur. Also removes obsolete image versions of typo3/core-testing-*.
 
     -h
         Show this help.
 
 Examples:
-    # Run unit tests using PHP 7.4
+    # Run all core unit tests using PHP 8.3
+    ./Build/Scripts/runTests.sh
```

Comment:
wouldn't this use PHP 7.4 and not 8.3? as 7.4 being default.

Reply:
correct. fixed.